homeassistant/components/sensor/zha.py

```diff
@@ -102,8 +102,8 @@ def unit_of_measurement(self):
     @property
     def state(self):
         """Return the state of the entity."""
-        if self._state == 'unknown':
-            return 'unknown'
+        if self._state == STATE_UNKNOWN:
```

Comment:
We shouldn't use `STATE_UNKNOWN` at all in the platforms or components. Probably change this in the zha component, that seems to set the initial state. Initial state should be `None`.